### PR TITLE
ENH: pipeline function to make a topography periodic by mirroring

### PR DESCRIPTION
**Pull request by AntoineSIMTEK**

### CODE REVIEW

File: SurfaceTopography/Uniform/Interpolation.py [hunk 1]
```diff
@@ -154,6 +155,48 @@ def interpolate_fourier(topography, nb_grid_pts):
...
+        nx, ny = self.nb_grid_pts
+        lnx, lny = self.nb_subdomain_grid_pts
+        sx, sy = self.physical_sizes
+        return np.meshgrid(
+            (self.subdomain_locations[0] + np.arange(lnx)) * sx / nx,
+            (self.subdomain_locations[1] + np.arange(lny)) * sy / ny,
+            indexing='ij')
+
+
+Topography.register_function("mirror_stitch_periodize",
```
Comment (pastewka) [inline SurfaceTopography/Uniform/Interpolation.py:198]: I would rename this to simply `mirror_stitch`.

**Review by pastewka — CHANGES_REQUESTED** [review]
Looks good, just one minor comments. Also, some tests appear to fail.

**Review by pastewka — APPROVED** [review]

**Comment by AntoineSIMTEK** [comment]
this branch fails because of problems with empty subdomains in muFFT. 

test_sphere with a communicator of length 4 is failing.

https://github.com/ComputationalMechanics/SurfaceTopography/pull/52/checks?check_run_id=1577117646

a fix is planned inside muFFT, correct @pastewka  ?

**Comment by pastewka** [comment]
Let's see if this passes. Do you want to update the requirements to muFFT 0.13.0?

**Comment by AntoineSIMTEK** [comment]
Actually besides of some number of MPI processes, the code is still compatible with the earlier muFFT. Only the test needs the newer  muFFT. 
So it depends what the philosphy behind "requirements" is.

**Review by pastewka — APPROVED** [review]

**Pull request merged**